utilities/unittests/test_sanity.py

```
@@ -684,3 +750,376 @@ def test_cluster_sanity_with_junitxml_property(
+        cluster_sanity(
+            request=mock_request,
+            admin_client=MagicMock(),
+            cluster_storage_classes_names=[],
+            nodes=MagicMock(),
+            hco_namespace=MagicMock(),
+        )
+
+        mock_check_webhook.assert_not_called()
+        mock_check_vm.assert_not_called()
+        # Should have warning about skipping webhook check
+        warning_calls = [call for call in mock_logger.warning.call_args_list]
```

Ruff (0.14.13)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)